### PR TITLE
fix: add migration to fix created on dsdocs

### PR DESCRIPTION
**Pull request by fontanierh**
## Description

context: https://dust4ai.slack.com/archives/C05B529FHV1/p1733213290583939

### CODE REVIEW

File: core/bin/migrations/20241203_fix_created_dsdocs.rs [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+                (
+                    ds_data,
+                    id,
+                    Document {
+                        data_source_id,
+                        created: created as u64,
+                        document_id,
+                        timestamp: timestamp as u64,
+                        tags: tags_array,
+                        parents,
```
Comment (spolu) [inline core/bin/migrations/20241203_fix_created_dsdocs.rs:129]: Why create a fully fledged document? vs just operate on docuemnt_id, hash  ?

**Review by spolu — APPROVED** [review]
LGTM assuming local tests with buggy GCS files

File: core/bin/migrations/20241203_fix_created_dsdocs.rs (hunk 1 above)
Comment (fontanierh) [reply core/bin/migrations/20241203_fix_created_dsdocs.rs:129]: No good reason tbh

Comment (spolu) [reply core/bin/migrations/20241203_fix_created_dsdocs.rs:129]: fine with it but makes it a bit harder to understand

Comment (fontanierh) [reply core/bin/migrations/20241203_fix_created_dsdocs.rs]: fixed

**Pull request merged**